### PR TITLE
[hr] minutes fix

### PR DESCRIPTION
**Pull request by kdcro101**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ node_modules/
 .DS_Store
 .idea
 *.swp
+.vscode
```
Comment (EvanHahn) [inline .gitignore]: Please remove this. I like it but would rather it be part of another pull request.

Comment (kdcro101) [reply .gitignore]: Ok

**Comment by kdcro101** [comment]
@EvanHahn ,

Regarding tests... 

No problem! I will commit it day after tomorow.

**Comment by kdcro101** [comment]
@EvanHahn 

Tests included + seconds fix

**Comment by EvanHahn** [comment]
Merged this manually in d68342869ffeac904ec8fdec6f3810da2a9512be and released this in `humanize-duration@3.15.1`.

_Hvala_ and good luck in the World Cup today!

**Comment by kdcro101** [comment]
Thank you for your wishes!

This is best result in Croatian history, now silver after bronze in 1998. nation is in ecstasy.
